CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
-* Refactor Struct value creation
+* Changed `LICENSE` to `Apache2.0`
```

Comment:
Looks like you have changed license from 'someone' to Apache2.0. But rather it's a technical issue to include license text instead of link.

Reply:
Fixed changelog record